mycroft/configuration/defaults/defaults.ini

```diff
@@ -92,3 +92,9 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[TwitterSkill]
+aToken = "access_token"
+aTokenSecret = "access_token_secret"
+cKey = "consumer_key"
+cSecret = "consumer_secret"
```

Comment:
Please, provide better name notation (and following the standard) for those keys: **access_token** instead of **aToken** and so on.